tests/functional/aws-node-sdk/test/multipleBackend/put.js

```diff
@@ -44,7 +46,13 @@ function awsGetCheck(objectKey, s3MD5, awsMD5, cb) {
         (err, res) => {
             assert.strictEqual(err, null, 'Expected success, got error ' +
             `on direct AWS call: ${err}`);
-            assert.strictEqual(res.ETag, `"${awsMD5}"`);
+            if (process.env.ENABLE_KMS_ENCRYPTION === 'true') {
+                // doesn't check ETag because it's different
+                // with every PUT with encryption
```

Comment:
AWS says it's not the md5 of the object data, but doesn't specify what it actually is